src/prefect/cli/auth.py

```diff
@@ -195,19 +208,20 @@ def switch_tenants(id, slug):
 
 @auth.command(hidden=True)
 @click.option("--name", "-n", required=True, help="A token name.", hidden=True)
-@click.option("--scope", "-s", required=True, help="A token scopre.", hidden=True)
-def create_token(name, scope):
+@click.option("--user-id", "-u", required=True, help="a user id.", hidden=True)
+def create_key(name, user_id):
```

Comment:
We'll have to maintain the old CLI command with a deprecation warning I think. At the very least, we'll need to keep the old command and throw an error saying what to use instead.

Reply:
Ah git diff is bad -- I see this still exists. I'll have to review it outside of GitHub.